### PR TITLE
Formats service period dates in 22-1995

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1329

image

### CODE REVIEW

File: src/js/edu-benefits/1995/components/ServicePeriodView.jsx [hunk 1]
```diff
@@ -1,10 +1,22 @@
 import React from 'react';
 
+/**
+ * Accepts YYYY-MM-DD and turns it into DD/MM/YYYY
+ *
+ * @param {String} date
+ * @return {String}
+ */
+function formatDisplayDate(date) {
```
Comment (jbalboni) [inline src/js/edu-benefits/1995/components/ServicePeriodView.jsx]: There's a function for this that's used in schemaform/review/widgets.jsx.

**Review by jbalboni — APPROVED** [review]
LGTM, just one comment.

File: src/js/edu-benefits/1995/components/ServicePeriodView.jsx (hunk 1 above)
Comment (cvalarida) [inline src/js/edu-benefits/1995/components/ServicePeriodView.jsx]: Ah, brilliant! I didn't see it before, so I just wrote a little regex. I'll reuse that.

**Comment by jbalboni** [comment]
@cvalarida Works for me.

**Pull request merged**